crates/rooch-types/src/crypto.rs

```diff
@@ -108,8 +110,8 @@ impl RoochKeyPair {
 
     pub fn private(&self) -> &[u8] {
         match self {
-            RoochKeyPair::Ed25519(kp) => kp.as_bytes(),
-            RoochKeyPair::Secp256k1(kp) => kp.as_bytes(),
+            RoochKeyPair::Ed25519(kp) => kp.as_ref(),
+            RoochKeyPair::Secp256k1(kp) => kp.secret.as_ref(),
```

Comment:
kp.as_bytes() == kp.as_ref() == kp.secret.as_ref()

It seems like we do not need to change this.

Reply:
> kp.as_bytes() == kp.as_ref() == kp.secret.as_ref()
> 
> It seems like we do not need to change this.

Yes. You're exactly right on this but when clicking on `as_bytes()` it will go to traits not the in-depth position to get insight of. However, I don't know if it's just me.

Reply:
The IDE can not locate the implemention.

Reply:
> The IDE can not locate the implemention.

I'm using VS-Code-like IDE and it will lead to the Private or Secret Key implementation to get the bytes.